lib/speaker_helpers.rb

```diff
@@ -0,0 +1,47 @@
+require 'httparty'
+require 'ostruct'
+
+module SpeakerHelpers
+  def all_speakers
+    data.speakers
+      .collect { |name, speaker_data| Speaker.new(name, speaker_data) }
+      .sort_by { |speaker| speaker.name }
```

Comment:
I wonder if we should order this by recency of the speakers talks, so it changes once each meeting is published to show the most recent speakers?  I don't think sorting by frequency of talks is a good idea, but maybe recency is weird.  We've enough speakers that maybe we'd want A..Z in page navigation so doing anything other than alphabetical is weird?

Reply:
I agree we should sort by most recently published, I think it's possible now I've figured out a way to pull the coverage data into a easier to work with format.